pkg/sync/rpc/server.go

```diff
@@ -321,10 +320,11 @@ func (s *SyncAgentServer) FilesSync(ctx context.Context, req *ptypes.FilesSyncRe
 			logrus.Errorf("Sync agent gRPC server failed to rebuild replica/sync files: %v", err)
 		} else {
 			s.RebuildStatus.State = types.ProcessStateComplete
+			logrus.Infof("Sync agent gRPC server finished rebuilding replica/sync files for replica %v", req.ToHost)
 		}
 		s.RebuildStatus.Unlock()
 
-		if err := s.FinishRebuild(); err != nil {
+		if err = s.FinishRebuild(); err != nil {
```

Comment:
Don't need to remove `:`. Just `return err` at the end.

Reply:
It's impossible to `return err` for this defer function.